docs/AreaTrigger.md

```diff
@@ -1,14703 +1,175 @@
-# AreaTrigger
+# areatrigger
 
-`Back-to:DBC`
+[<-Back-to:World](database-world.md)
 
-**AreaTrigger.dbc**
+**The \`areatrigger\` table**
```

Comment:
That's what I wanted to point out in the discord

Check here, the backticks are shown https://github.com/azerothcore/wiki/blob/9c416ce7b31686fa7ee5795f2e6475a51d0f50dc/docs/AreaTrigger.md

Reply:
They should be shown though? Or am I misunderstanding you completely?

Reply:
no they shouldn't, why should they?? I don't see the point

`table` is fine

but with the backticks shown I don't see any interest